docs/security.html

```diff
@@ -47,7 +47,7 @@ <h3 class="anchor-heading"><a id="security_ssl" class="anchor-link"></a><a href=
             The tool supports two different keystore formats, the Java specific jks format which has been deprecated by now, as well as PKCS12.
             PKCS12 is the default format as of Java version 9, to ensure this format is being used regardless of the Java version in use all following
             commands explicitly specify the PKCS12 format.
-            <pre class="line-numbers"><code class="language-bash">keytool -keystore {keystorefile} -alias localhost -validity {validity} -genkey -keyalg RSA -storetype pkcs12</code></pre>
+            <pre class="line-numbers"><code class="language-bash">&gt; keytool -keystore {keystorefile} -alias localhost -validity {validity} -genkey -keyalg RSA -storetype pkcs12</code></pre>
```

Comment:
I am wondering if it is a good idea to add `>` in front of the commands. It is something additional users need to remove when they copy & paste the commands.

Reply:
Now I saw that it has been already used further down. I fine with keeping it to distingiush better between configs and commands. However, I found some places where `>` is still missing (see further down).

Reply:
The examples present in the documentation, use more frequently the `>` character when showing commands. Sometimes a `$` is shown, and really occasionally nothing is put in there.
As `>` was the most common used one already in the codebase, I decided to unify on this character.